string eqp_cd
int8   dx_direct          # 0:정방향,1:역방향
int8   dy_direct          # 0:정방향,1:역방향
int32  dz_mm
string cr_op_indi_id
int32  plate_top_len 
int32  plate_max_width
bool   lidar_use
bool   camera_use
string raw
builtin_interfaces/Time stamp
